app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+
+object FontScaleConfigurationUtil {
+  fun adjustFontSize(
+    context: Context,
+    result: StoryTextSize
+  ) {
+    val configuration = context.resources.configuration
+    configuration.fontScale = getStoryTextSize(result)
+    val metrics: DisplayMetrics = context.getResources().getDisplayMetrics()
+    val wm =
+      context.getSystemService(WINDOW_SERVICE) as WindowManager?
+    wm!!.defaultDisplay.getMetrics(metrics)
+    metrics.scaledDensity = configuration.fontScale * metrics.density
+    context.createConfigurationContext(configuration)
+    context.resources.displayMetrics.setTo(metrics)
+  }
+
+  fun getStoryTextSize(storyTextSize: StoryTextSize): Float {
```

Comment:
Is this actually returning a size multiplier? Suggest ``computeStoryTextSizeMultiplier`` since it's also computing a value rather than getting one.